spec/factories/user_factory.rb

```diff
@@ -57,6 +57,18 @@
       end
     end
 
+    trait :with_roomLimit_100_permission do
+      after(:create) do |user|
+        create(:role_permission, role: user.role, permission: create(:permission, name: 'RoomLimit'), value: '100')
+      end
+    end
+
+    trait :with_roomLimit_3_permission do
+      after(:create) do |user|
+        create(:role_permission, role: user.role, permission: create(:permission, name: 'RoomLimit'), value: '3')
+      end
+    end
```

Comment:
I wouldn't put this in a trait, I would just create the role permission directly in the rooms_controller spec